created new video-export page (#31)
iXtenda/iVP-gitbook: home/ivp-planning/advanced-tools/video-export-tool.md

# Video Export

With update 2307.0 it is now possible to directly export your recordings via the "Video Export" option, found in the "File" drop-down menu in the top left of the program.

![](../../../.gitbook/assets/video-export.png)  

> Once you have clicked on the "Video Export" option, a new window will open, in which you can make changes to your recording.  

Reviewer: Regarding the writing style of this documentation, please consider the following:

1. Avoid using past time
2. Keep sentences short and simple/precise

```suggestion
Click on "Video Export" to open a new window that allows you to configure your recording.
```

> Once you have clicked on the checkmark the program will start recording your video, and save it in the iVP-recordings folder.

Reviewer: Same here as above, avoid past time and keep sentences short and simple.

# Video Export
  
It is possible to directly export your recordings via the "Video Export" option, found in the "File" drop-down menu in the top left of the application.

![](../../../.gitbook/assets/video-export.png)  

Click on "Video Export" to open a new window that allows you to configure your recording.

> When ypou click on the checkmark the program will start recording your video, and save it in the iVP-recordings folder.

Reviewer: ```suggestion
When you click on the checkmark Planning will start recording your video, and save it in the iVP-recordings folder.
```
